VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -65,9 +85,33 @@ def device_list_for_billing(self):
         devices_to_bill = [] #list to store device names to be billed
         device_items = disk_util.get_device_items(None)
         for device_item in device_items :
-            # self.logger.log("Device name : {0} ".format(str(device_item.name)),True)
+            self.logger.log("Device name : {0} ".format(str(device_item.name)),True)
             if str(device_item.name).startswith("sd"):
                 devices_to_bill.append("/dev/{0}".format(str(device_item.name)))
+        self.logger.log("Initial billing items {0}".format(devices_to_bill))
+        self.logger.log("lsscsi_list {0}".format(self.lsscsi_list))
+        if(len(self.lsscsi_list)!=0):
+            for i in self.lsscsi_list:
+                self.included=True
+                idxOfColon=i.rindex(':',0,i.index(']'))#to get the index of last ':'
+                idxOfColon+=1
+                lunNumbers=int(i[idxOfColon:i.index(']')])
+                self.temp=i.split()
```

Comment:
NIT: In general, it's a good practice to use a space before and after using binary operators such as '='. Across the code